pages/scenario_selection_page.py

```diff
@@ -0,0 +1,310 @@
+    """
+
+    st.markdown("# Lifetime costs: scenario selection")
+
+    st.markdown("### Select a predefined scenario or build your own")
+    st.write(
+        """
+             If you select a predefined scenario, the parameters will be set automatically. If you choose to build a custom scenario, start by naming it.
+             Below you can also select the installation year and cost decile for your results. When you build a custom scenario, you can then define other the parameters in more detail.
+             """
+    )
+    scenario_names = [scenarios[key]["name"] for key in scenarios.keys()]
+    scenario_names = scenario_names + ["Build a custom scenario"]
+
+    scenario_select_col, new_scenario_name_col = st.columns(2)
+
+    with scenario_select_col:
+        selected_scenario = st.selectbox(
+            label="Select a scenario",
+            options=scenario_names,
+            index=0,
+            key="selected_scenario",
+        )
+
+    with st.expander("📅 Expand to select installation year and cost decile"):
+        installation_year_col, decile_col = st.columns(2)
+
+        with installation_year_col:
+            installation_year = st.slider(
+                label="Select the installation year",
+                min_value=2025,
+                max_value=2035,
+                value=2025,
+                step=1,
+                key="custom_installation_year_input",
+                help="The year the heating system is installed",
+            )
+
+        with decile_col:
+            cost_decile = st.selectbox(
+                label="Select the cost decile (50 corresponds to the median)",
+                options=range(10, 100, 10),
+                index=4,
+                key="income_decile_input",
+                help="Select the cost decile, where 50 corresponds to the median.",
+            )
+
+    if selected_scenario == "Build a custom scenario":
+        with new_scenario_name_col:
+            scenario_name = st.text_input(
+                label="Name your custom scenario",
+                value="",
+                key="custom_scenario_name_input",
+            )
+
+        if scenario_name != "":
+            with st.expander("⚙️ Define your custom scenario parameters"):
+                st.write(
+                    "Expand the sections below to define your custom scenario parameters."
+                )
+                with st.expander("👨🏻‍🔧 Maintenance costs"):
+                    ashp_maintenance_cost_col, boiler_maintenance_cost_col = st.columns(
+                        2
+                    )
+
+                    with ashp_maintenance_cost_col:
+                        ashp_maintenance_cost = st.slider(
+                            label="ASHP annual maintenance cost (£)",
+                            min_value=0,
+                            max_value=500,
+                            value=config.maintenance_costs_default["ashp"],
+                            step=10,
```

Comment:
```suggestion
                            step=5,
```

I'm leaning towards more granularity here and maybe reducing step size from 10 to 5? If that makes the slider too fiddly, then maybe also reducing the max value from 500 to like 300 or 350?

Based on the changes to the maintenance cost computation arguments, I'm assuming we will also have another slider for each heating system for frequency of maintenance (another slider 0 to 4 [quarterly], with 0.2 [=every 5 years] step size maybe?)